build_release_archive.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# This script creates archive files that can be distributed as a release. The
+# files that github provides do not include files stored in git lfs, so this
+# process is required to create working archives that we can publish on github.
+# The resulting archives will be placed in the current directory.  Because this
+# script downloads and packages a large repo it may take a long time to run.
+
+# This script works under /tmp and it assumes that previous artifacts under /tmp
+# (such as previously downloaded repos or previously created archives) can be
+# safely deleted.
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <committish>"
+  echo "where <committish> is a commit, branch, or tag that defines the release"
+  echo "to be archived."
+  exit 1
+fi
+
+REPO_NAME="earthenterprise"
+CLONE_DIR="/tmp"
+
+TAR_FILE=${REPO_NAME}.tar.gz
+ZIP_FILE=${REPO_NAME}.zip
+RUN_DIR=`pwd`
+
+# Download the repository
+cd ${CLONE_DIR}
+rm -Rf ${REPO_NAME}
+git clone git@github.com:google/${REPO_NAME}.git
```

Comment:
Suggestion: Use `git clone -b <branch_name> --depth 1 <repository_url>` instead of checking out the whole history on all branches.  I think it saves you downloading about a GB.